exercises/practice/square-root/.meta/proof.ci.wren

```diff
@@ -0,0 +1,13 @@
+class SquareRoot {
+
+  static squareRoot(input) {
+    var candidate = 1
+
+    if (input >= 2) {
+      while ((candidate * candidate) < input) candidate = candidate + 1
```

Comment:
```suggestion
  static squareRoot(n) {
    var candidate = 1

    if (n >= 2) {
      while ((candidate * candidate) < n) candidate = candidate + 1
```

Reply:
`input` is such a vague name... `number` or `n`( one of the few one letter variables I think is ok) is far better.